packages/realm_dart/test/baas_helper.dart

```diff
@@ -148,7 +140,7 @@ class BaasHelper {
   Future<void> _waitForInitialSync(BaasApp app) async {
     while (true) {
       try {
-        print('Validating initial sync is complete...');
+        print('Validating initial sync for $app is complete...');
```

Comment:
Should we log instead?

Reply:
We could here, but we're using print statements in the rest of the baas setup code, so I felt it's fine to keep it in. Also, the way we've setup the logger is to only log on failure, while I think those one-time logs are useful anyway.